src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -47,6 +47,12 @@
 import java.util.stream.Stream;
 
 public class GenotypeGVCFsIntegrationTest extends CommandLineProgramTest {
+    //GenotypeGVCFs should work on input derived from:
```

Comment:
In the integration test class?

Reply:
As a reminder to include coverage.  Where else?